tests-cpp/test_1494-layout-builder.cpp

```diff
@@ -1154,7 +1154,6 @@ void
 test_Regular_size0() {
   RegularBuilder<0, NumpyBuilder<double>> builder;
 
-  auto& subbuilder = builder.begin_list();
```

Comment:
Oh, you're right. This `begin_list` is potentially functional, but in practice it's not (hence, no error). I should not have removed it, though I can leave out the `subbuilder`, since it's not used.